[2024-11-18T01:12:32.928+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T01:12:32.955+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_api_sales_data.test manual__2024-11-18T01:12:25.508555+00:00 [queued]>
[2024-11-18T01:12:32.967+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_api_sales_data.test manual__2024-11-18T01:12:25.508555+00:00 [queued]>
[2024-11-18T01:12:32.968+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-18T01:12:32.992+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): test> on 2024-11-18 01:12:25.508555+00:00
[2024-11-18T01:12:33.002+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=93) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-18T01:12:33.005+0000] {standard_task_runner.py:72} INFO - Started process 102 to run task
[2024-11-18T01:12:33.005+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'raw_api_sales_data', 'test', 'manual__2024-11-18T01:12:25.508555+00:00', '--job-id', '262', '--raw', '--subdir', 'DAGS_FOLDER/fetch_data.py', '--cfg-path', '/tmp/tmp1pi0a56o']
[2024-11-18T01:12:33.007+0000] {standard_task_runner.py:105} INFO - Job 262: Subtask test
[2024-11-18T01:12:33.070+0000] {task_command.py:467} INFO - Running <TaskInstance: raw_api_sales_data.test manual__2024-11-18T01:12:25.508555+00:00 [running]> on host 1c1ea357472a
[2024-11-18T01:12:33.185+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_api_sales_data' AIRFLOW_CTX_TASK_ID='test' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T01:12:25.508555+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-18T01:12:25.508555+00:00'
[2024-11-18T01:12:33.187+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-18T01:12:33.188+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-18T01:12:33.188+0000] {logging_mixin.py:190} INFO - Current task name:test state:running start_date:2024-11-18 01:12:32.956005+00:00
[2024-11-18T01:12:33.189+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data and current dag run status:running
[2024-11-18T01:12:33.190+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T01:12:33.204+0000] {base.py:84} INFO - Retrieving connection 'postgres_default'
[2024-11-18T01:12:33.207+0000] {logging_mixin.py:190} INFO - Error: connection to server at "localhost" (::1), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-11-18T01:12:33.208+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-18T01:12:33.224+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-18T01:12:33.225+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=raw_api_sales_data, task_id=test, run_id=manual__2024-11-18T01:12:25.508555+00:00, execution_date=20241118T011225, start_date=20241118T011232, end_date=20241118T011233
[2024-11-18T01:12:33.240+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-18T01:12:33.241+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-18T01:12:33.242+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data queued_at:2024-11-18 01:12:25.524757+00:00
[2024-11-18T01:12:33.242+0000] {logging_mixin.py:190} INFO - Task hostname:1c1ea357472a operator:PythonOperator
[2024-11-18T01:12:33.261+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-18T01:12:33.298+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-18T01:12:33.301+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
